docs/sdk.rst

```diff
@@ -0,0 +1,191 @@
+SDK
+============
+
+The **funcX SDK** provides a programmatic interface to funcX from Python.
+The SDK provides a convenient Pythonic interface to:
+
+1. Register functions
+2. Register containers and execution environments
+3. Launch registered function on accessible endpoints
+4. Check the status of launched functions and
+5. Retrieve outputs from functions
+
+The SDK provides a client class for interacting with funcX. The client
+abstracts authentication and provides an interface to make funcX
+API calls without needing to know the funcX REST endpoints for those operations.
+You can instantiate a funcX client as follows.
+
+.. code-block:: python
+
+  from funcx.sdk.client import FuncXClient
+  fxc = FuncXClient()
+
+Instantiating a client will start an authentication process where you will be asked to authenticate via Globus Auth.
+We require every interaction with funcX to be authenticated as it enables enforced
+access control on both functions and endpoints.
+Globus Auth is an identity and access management platform that provides authentication brokering
+capablities enabling users to login using one of several hundred supported identities.
+It also provides group and profile management for user accounts.
+As part of the authentication process, funcX will request access
+to your identity (to retrieve your email address) and Globus Groups. funcX uses
+Groups to facilitate sharing and to make authorization decisions.
+funcX allows endpoints and functions to be shared by associating a Globus Group.
+
+.. note:: funcX internally caches function, endpoint, and authorization lookups. Caches are based on user authentication tokens. To force refresh cached
+          entries you can re-authenticate your client with `force_login=True`.
```

Comment:
```suggestion
          entries, you can re-authenticate your client with `force_login=True`.
```

Reply:
Assuming this is `fxc = FuncXClient(force_login=True)`, maybe provide this code?